packages/explorer/package.json

```
@@ -64,5 +64,8 @@
       "optional": true
     }
   },
-  "dependencies": {}
+  "dependencies": {
+    "graphql-ws": "^5.9.0",
+    "subscriptions-transport-ws": "^0.11.0"
```

```suggestion
    "subscriptions-transport-ws": "^0.11.0",
    "use-deep-compare-effect": "^1.8.1"
```

My embedded-explorer-test app was getting this error: `Module not found: Error: Can't resolve 'use-deep-compare-effect' in '/Users/william/Apollo/test/embedded-explorer-test/node_modules/@apollo/explorer/react'`. Should we add this or do we want users to have to `npm i use-deep-compare-effect`?

---

its optional! you only need it if you use the react one - so we do want folks to opt in